src/libslic3r/PrintObject.cpp

```diff
@@ -1989,12 +1988,20 @@ void PrintObject::bridge_over_infill()
     };
 
     std::map<size_t, std::vector<CandidateSurface>> surfaces_by_layer;
+    // Ioannis Giannakas:
```

Comment:
Hi Ioannis,
May I request that we stick to the convention of the `// Orca:` format?
Just so that it's easier for maintenance.

Reply:
Of course!!!

Reply:
done